facia-tool/build.sbt

```diff
@@ -20,3 +20,5 @@ javaOptions in Universal ++= Seq(
     "-J-XX:+PrintGCDateStamps",
     s"-J-Xloggc:/var/log/${packageName.value}/gc.log"
 )
+
+ivyScala := ivyScala.value map { _.copy(overrideScalaVersion = true) }
```

Comment:
Added this to resolve compilation errors which occurred as a result of upgrading panda, but it turned out the fix was to upgrade sbt. Can remove.